src/ui/components/DefinitionList/index.js

```diff
@@ -0,0 +1,50 @@
+/* @flow */
+import * as React from 'react';
+import makeClassName from 'classnames';
+
+import './styles.scss';
+
+
+type DefinitionItemProps = {|
+  children: React.Node,
+  className?: string,
+  title: React.Node,
+|};
+
+export const DefinitionItem = (
+  { children, className = '', title }: DefinitionItemProps = {}
+) => {
+  return (
+    <React.Fragment>
+      <dt
+        className={makeClassName(className, {
+          [`${className}-title`]: className && className.length,
```

Comment:
I think we should avoid dynamic class names derived from the configured class name because there is no way to look at the calling code and guess that it's happening.

From what I can tell, there are no `AddonMoreInfo...title` classes in any style sheets so I say you either remove it or replace it with a hard-coded class like `DefinitionItem-title`.

If you really needed a custom class like this then it would be better as a custom property, e.g.:

```js
          <DefinitionItem
            className="AddonMoreInfo-version"
            title={i18n.gettext('Version')}
            titleClassName="AddonMoreInfo-version-title"
          >
            {version}
          </DefinitionItem>
```

When looking at the component instance, the custom styling possibilities would be more obvious.

However, I say you just remove the title class for now since we don't need it.

Reply:
That's a good point. In the past they were useful for testing but I don't think that's the case anymore, so works for me.